covid19-belgium.csv

```diff
@@ -30,4 +30,5 @@
 29/03/2020,,1702,,10836,4138,867,78,431,296,1359,906,691,85,6381,3187,1065,,,
 30/03/2020,,1063,,11899,4524,927,82,513,168,1527,683,164,198,7064,3351,1263,,,
 31/03/2020,2500,876,52072,12775,4920,1021,98,705,168,1696,467,203,189,7531,3554,1452,,,
-01/04/2020,4255,1189,56327,13964,4995,1088,123,828,436,2132,688,349,128,8219,3903,1580,,,
\ No newline at end of file
+01/04/2020,4255,1189,56327,13964,4995,1088,123,828,436,2132,688,349,128,8219,3903,1580,,,
+02/04/2020,"",1384,"",15348,6716,1044,183,1011,363,2495,912,317,137,9131,4220,1717,"","",""
```

Comment:
Could you remove the double quotes you used for empty values ? The rest looks good to me